specs/src/specs/public_key_cryptography.md

```diff
@@ -0,0 +1,23 @@
+# Public-Key Cryptography
+
+<!-- toc -->
+
+Consensus-critical data is authenticated using [ECDSA](https://www.secg.org/sec1-v2.pdf), with the curve [secp256k1](https://en.bitcoin.it/wiki/Secp256k1). A highly-optimized library is available in C (<https://github.com/bitcoin-core/secp256k1>), with wrappers in Go (<https://pkg.go.dev/github.com/ethereum/go-ethereum/crypto/secp256k1>) and Rust (<https://docs.rs/crate/secp256k1>).
```

Comment:
Just noting that this file and section makes it sounds as if ecdsa/secp was the only pub-key crypto we were using. Validators still use ed25519 during consensus (which is consensus critical data).

Reply:
Both signatures schemes should be described then.

Reply:
Agreed will add a section for ed25519 based on https://github.com/cometbft/cometbft/blob/be6f4272d4ebedf264e3ce29475ffd31ecadff47/spec/core/encoding.md?plain=1#L42-L66

Reply:
Added a section in https://github.com/celestiaorg/celestia-app/pull/1553/commits/64164c41784b508da778bd630c2f3c3db1362695